specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-05-01-preview/containerregistry_scopemap.json

```diff
@@ -0,0 +1,916 @@
+            }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ScopeMap"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ScopeMapCreate": {
+            "$ref": "./examples/ScopeMapCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "ScopeMaps"
+        ],
+        "description": "Deletes a scope map from a container registry.",
+        "operationId": "ScopeMaps_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeMapNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "202": {
+            "description": "The request was successful; the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "The webhook does not exist in the subscription."
+          }
+        },
+        "x-ms-examples": {
+          "ScopeMapDelete": {
+            "$ref": "./examples/ScopeMapDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "ScopeMaps"
+        ],
+        "description": "Updates a scope map with the specified parameters.",
+        "operationId": "ScopeMaps_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeMapNameParameter"
+          },
+          {
+            "name": "scopeMapUpdateParameters",
+            "in": "body",
+            "description": "The parameters for updating a scope map.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ScopeMapUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/ScopeMap"
+            }
+          },
+          "201": {
```

Comment:
A 202 would be more accurate

Reply:
for PUT /PATCH resource we are following the AzureAsync header pattern rather than Location header pattern as recommended by https://armwiki.azurewebsites.net/faq/Intro.html?q=async